forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -167,10 +173,13 @@ class ForageSDK : ForageSDKInterface {
      *
      *         when (response) {
      *             is ForageApiResponse.Success -> {
-     *                 // response.data will have a .snap and a .cash value
+     *                 val balance = response.toBalance()
+     *                 val ebtBalance = balance as EbtBalance
```

Comment:
Similar question here for `balance as EbtBalance` as I have in [this comment](https://app.graphite.dev/github/pr/teamforage/forage-android-sdk/249/FX-1332-Typed-SDK-response-helpers?panel=files#comment-PRRC_kwDOIQqcBs5e60mt).